codeview/src/main/java/io/github/kbiakov/codeview/adapters/CodeWithDiffsAdapter.kt

```diff
@@ -11,16 +13,18 @@ import io.github.kbiakov.codeview.views.LineDiffView
  *
  * @author Kirill Biakov
  */
-class CodeWithDiffsAdapter : AbstractCodeAdapter<DiffModel> {
+open class CodeWithDiffsAdapter : AbstractCodeAdapter<DiffModel> {
+
+    constructor(context: Context, h: Highlighter) : super(context, h)
+
     /**
      * Default constructor.
      */
-    constructor(context: Context, content: String) : super(context, content)
+    constructor(context: Context, content: String, theme: ColorThemeData) : super(context, content, theme)
```

Comment:
Theme is not necessary param (with the rest, which are not presented). Init with default theme.